resources/metadata/gradleExecuteBuild.yaml

```diff
@@ -115,6 +115,36 @@ spec:
           - STAGES
           - PARAMETERS
         default: false
+      - name: applyCreateBOMForAllProjects
```

Comment:
Similar comment as `applyPublishingForAllProjects` , maybe we can work with `excludeCreateBOMForProjects` only?